### PR TITLE
Improve ObserverRunner

### PR DESCRIPTION
**Pull request by xitronix**
That's PR reduce wallet's test time too

### CODE REVIEW

File: universal-login-sdk/lib/core/observers/ObserverRunner.ts [hunk 1]
```diff
@@ -28,10 +29,11 @@ abstract class ObserverRunner {
 
   stop() {
     this.state = 'stop';
+    clearTimeout(this.timeout);
   }
 
   async finalizeAndStop() {
-    this.state = 'stopping';
+    this.state = this.isRunning() ? 'stopping' : 'stop';
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/core/observers/ObserverRunner.ts]: This is probably a bug, i.e.:
`if (state == 'stopping')` `finalizeAndStop()` will change it to `stop` while it should not.

Comment (xitronix) [reply universal-login-sdk/lib/core/observers/ObserverRunner.ts]: Implementation of isRunnning bellow: 
``` ts 
protected isRunning() {
    return this.state !== 'stop';
 }
```
So if state === 'stopping' finalizeAndStop `isRunning()` will be true, so finalizeAndStop will set state to 'stopping'

Comment (marekkirejczyk) [reply universal-login-sdk/lib/core/observers/ObserverRunner.ts]: Ok, so it is a bad getter. IsRunning should returninfo if observer is in state running not running or stopping

Comment (xitronix) [reply universal-login-sdk/lib/core/observers/ObserverRunner.ts]: So I will update it and change condition

**Pull request merged**